server.py

```diff
@@ -0,0 +1,38 @@
+################################################################################
+#                                                                              #
+# This file implements the http server and the handler of http requests        #
+#                                                                              #
+################################################################################
+
+# http.server and socketserver are imported to implement the http server
+import http.server
+import socketserver
+# extenso imported to write down the number
+import extenso
+
+# this is the custom handler that 
+class Handler(http.server.BaseHTTPRequestHandler):
+    # the only thing that needs to be modified is create the do_GET method 
+    def do_GET(self):
+        self.send_response(200)
+        self.send_header("Content-type", "json")

```

Comment:
typo, é com `T` maiúsculo

Reply:
Obrigado, não tinha verificado os headers de HTTP, farei a correção.

```suggestion
        self.send_header("Content-Type", "json")
```